[UP-216] Fix disable hotwallet for algorand

makarychev opened the pull request:
[UP-216 Fix Robowallet withdrawals for Algorand projects (ALGO, NOTE)](https://comakery-upside.atlassian.net/browse/UP-216)

Here is resolved 2 issues:
1. Coin & token balance is caching and it could led to outdated data usage - removed caching
2. Purestake node updates balance with ~5-6 seconds timeout which is critical on disableWallet when we need to calculate exact amount for withdrawal - withdraw 0 amount and what left is sent to Remainder account which the same in our case

Alexey1100 commented on lib/blockchains/AlgorandBlockchain.js:
```diff
@@ -369,14 +365,15 @@ class AlgorandBlockchain {
+      // wait timeout in order to not get too many requests error and let nodes to sync balance after previous tx
+      await new Promise(resolve => setTimeout(resolve, this.envs.algorandDisableWalletCoinBalanceReqTimeout * 1000))
+      const coinBalance = await this.getAlgoBalanceForHotWallet(hotWallet.address)
+
```
I think we can avoid the issue by using `CloseRemainderTo` field, which enforces to send all of the remaining coins to specified address:

<img width="1158" alt="Screenshot 2023-01-27 at 18 54 33" src="https://user-images.githubusercontent.com/10757562/215146641-67913eee-aa59-417f-b614-91112ade09fc.png">


https://developer.algorand.org/docs/get-details/transactions/transactions/#payment-transaction

https://www.algorand.foundation/news/limited-key-rotation

makarychev replied:
Thanks for idea. Inside `withdrawCoinsTx` we specify identical Receiver and CloseRemainderTo, so I think the amount must be less than available. I will try to set 0 amount and check

makarychev commented:
Added minimum Algorand account balance for `enoughCoinBalanceToSendTransaction` calculation.
Removed additional env and close account in any case in order to get reserved minimum Algorand account balance. For that purpose send 0 amount to recipient and another part to remainder account which is the same in our case
